service/src/main/java/uk/gov/hmcts/reform/fpl/service/RespondentService.java

```diff
@@ -84,4 +88,28 @@ public AboutToStartOrSubmitCallbackResponse addHiddenValues(CaseDetails caseDeta
             .data(data)
             .build();
     }
+
+    public String buildRespondentLabel(List<Element<Respondent>> respondents) {
+        StringBuilder sb = new StringBuilder();
+
+        if (isNotEmpty(respondents)) {
+            for (int i = 0; i < respondents.size(); i++) {
```

Comment:
I don't believe you actually refactored that <3 ❤️

Reply:
🤢 🤢 🤢